check_stability.py

```diff
@@ -402,6 +407,14 @@ def process_results(log, iterations):
     return results, inconsistent
 
 
+def markdown_adjust(s):
+    s = s.replace('\t', u'\u2409')
```

Comment:
I'm not sure these first three aren't confusing. I guess to do this correctly we would need real table markup. In the interim replacing them with `\\t` etc. could be better.